Makefile

```diff
@@ -535,9 +543,10 @@ show-usage: show-header
 	@printf "\tdefault             : same as just \"make\".\n"
 	@printf "\tgenerate-config     : create configuration file.\n"
 	@printf "\tinstall             : install files [2].\n"
-	@printf "\tinstall-$(CC_TYPE)-system   : install using standard $(CC_PROJECT_NAME) system paths\n"
-	@printf "\tinstall-$(KATA_TYPE)-system : install using standard $(KATA_PROJECT_NAME) system paths\n"
-	@printf "\tshow-summary        : show install locations\n"
+	@printf "\tinstall-$(CC_TYPE)-system   : install using standard $(CC_PROJECT_NAME) system paths.\n"
+	@printf "\tinstall-$(KATA_TYPE)-system : install using standard $(KATA_PROJECT_NAME) system paths.\n"
```

Comment:
sneaky '.'s threw me for a moment ;-)

Reply:
They *are* on a different commit though ;)